tests/extension/oneapi_kernel_compiler_spirv/kernel_compiler_spirv.cpp

```diff
@@ -55,14 +58,13 @@ void testSimpleKernel(sycl::queue& q, const sycl::kernel& kernel,
 
   constexpr int N = 4;
   std::array<int, N> input_array{0, 1, 2, 3};
-  std::array<int, N> output_array{};
 
-  sycl::buffer input_buffer(input_array.data(), sycl::range<1>(N));
-  sycl::buffer output_buffer(output_array.data(), sycl::range<1>(N));
+  sycl::buffer<int> input_buffer{input_array.data(), sycl::range<1>(N)};
+  sycl::buffer<int> output_buffer{sycl::range<1>(N)};
 
   q.submit([&](sycl::handler& cgh) {
-    cgh.set_arg(0, input_buffer.get_access<sycl::access::mode::read>(cgh));
-    cgh.set_arg(1, output_buffer.get_access<sycl::access::mode::write>(cgh));
+    cgh.set_args(input_buffer.get_access<sycl::access::mode::read>(cgh),
+                 output_buffer.get_access<sycl::access::mode::write>(cgh));
```

Comment:
Since these are new tests, probably worth to use `sycl::accessor { ... }` API instead of old `.get_access` syntax.